dotnet/VaccinationSample failed to build: program.cs syntax error

To reproduce...
```
C:\TDW\repos\sdk-examples\dotnet\VaccinationSample>dotnet restore
  Determining projects to restore...
  All projects are up-to-date for restore.

C:\TDW\repos\sdk-examples\dotnet\VaccinationSample>dotnet run
C:\TDW\repos\sdk-examples\dotnet\VaccinationSample\Program.cs(9,42): error CS1003: Syntax error, ',' expected [C:\TDW\repos\sdk-examples\dotnet\VaccinationSample\VaccinationSample.csproj]
C:\TDW\repos\sdk-examples\dotnet\VaccinationSample\Program.cs(10,21): error CS1003: Syntax error, ',' expected [C:\TDW\repos\sdk-examples\dotnet\VaccinationSample\VaccinationSample.csproj]

The build failed. Fix the build errors and run again.

C:\TDW\repos\sdk-examples\dotnet\VaccinationSample>
```


Reference: https://github.com/mwherman2000/sdk-examples/blob/main/dotnet/VaccinationSample/Program.cs

Adding the 2 missing commas allows the build to proceed but it still fails with these messages...
```
Build FAILED.

C:\TDW\repos\sdk-examples\dotnet\VaccinationSample\Program.cs(8,20): error CS0246: The type or namespace name 'ServiceOptions' could not be found (are you missing a using directive or an assembly reference?) [C:\TDW\repos\sdk-examples\dotnet\VaccinationSample\VaccinationSample.csproj]
C:\TDW\repos\sdk-examples\dotnet\VaccinationSample\Program.cs(14,26): error CS0246: The type or namespace name 'AccountService' could not be found (are you missing a using directive or an assembly reference?) [C:\TDW\repos\sdk-examples\dotnet\VaccinationSample\VaccinationSample.csproj]
C:\TDW\repos\sdk-examples\dotnet\VaccinationSample\Program.cs(15,38): error CS1061: 'ProviderService' does not contain a definition for 'CreateEcosystem' and no accessible extension method 'CreateEcosystem' accepting a first argument of type 'ProviderService' could be found (are you missing a using directive or an assembly reference?) [C:\TDW\repos\sdk-examples\dotnet\VaccinationSample\VaccinationSample.csproj]
C:\TDW\repos\sdk-examples\dotnet\VaccinationSample\Program.cs(15,6): error CS8130: Cannot infer the type of implicitly-typed deconstruction variable 'ecosystem'. [C:\TDW\repos\sdk-examples\dotnet\VaccinationSample\VaccinationSample.csproj]
C:\TDW\repos\sdk-examples\dotnet\VaccinationSample\Program.cs(15,17): error CS8130: Cannot infer the type of implicitly-typed deconstruction variable '_'. [C:\TDW\repos\sdk-examples\dotnet\VaccinationSample\VaccinationSample.csproj]
C:\TDW\repos\sdk-examples\dotnet\VaccinationSample\Program.cs(15,17): error CS8183: Cannot infer the type of implicitly-typed discard. [C:\TDW\repos\sdk-examples\dotnet\VaccinationSample\VaccinationSample.csproj]
C:\TDW\repos\sdk-examples\dotnet\VaccinationSample\Program.cs(33,30): error CS0246: The type or namespace name 'CredentialsService' could not be found (are you missing a using directive or an assembly reference?) [C:\TDW\repos\sdk-examples\dotnet\VaccinationSample\VaccinationSample.csproj]
C:\TDW\repos\sdk-examples\dotnet\VaccinationSample\Program.cs(41,15): error CS1061: 'WalletService' does not contain a definition for 'Options' and no accessible extension method 'Options' accepting a first argument of type 'WalletService' could be found (are you missing a using directive or an assembly reference?) [C:\TDW\repos\sdk-examples\dotnet\VaccinationSample\VaccinationSample.csproj]
C:\TDW\repos\sdk-examples\dotnet\VaccinationSample\Program.cs(44,50): error CS0103: The name 'VaccinationCertificateUnsigned' does not exist in the current context [C:\TDW\repos\sdk-examples\dotnet\VaccinationSample\VaccinationSample.csproj]
C:\TDW\repos\sdk-examples\dotnet\VaccinationSample\Program.cs(61,15): error CS1061: 'WalletService' does not contain a definition for 'Options' and no accessible extension method 'Options' accepting a first argument of type 'WalletService' could be found (are you missing a using directive or an assembly reference?) [C:\TDW\repos\sdk-examples\dotnet\VaccinationSample\VaccinationSample.csproj]
C:\TDW\repos\sdk-examples\dotnet\VaccinationSample\Program.cs(63,34): error CS1061: 'WalletService' does not contain a definition for 'InsertItemAsync' and no accessible extension method 'InsertItemAsync' accepting a first argument of type 'WalletService' could be found (are you missing a using directive or an assembly reference?) [C:\TDW\repos\sdk-examples\dotnet\VaccinationSample\VaccinationSample.csproj]
C:\TDW\repos\sdk-examples\dotnet\VaccinationSample\Program.cs(64,39): error CS1061: 'WalletService' does not contain a definition for 'SearchAsync' and no accessible extension method 'SearchAsync' accepting a first argument of type 'WalletService' could be found (are you missing a using directive or an assembly reference?) [C:\TDW\repos\sdk-examples\dotnet\VaccinationSample\VaccinationSample.csproj]
C:\TDW\repos\sdk-examples\dotnet\VaccinationSample\Program.cs(74,15): error CS1061: 'WalletService' does not contain a definition for 'Options' and no accessible extension method 'Options' accepting a first argument of type 'WalletService' could be found (are you missing a using directive or an assembly reference?) [C:\TDW\repos\sdk-examples\dotnet\VaccinationSample\VaccinationSample.csproj]
C:\TDW\repos\sdk-examples\dotnet\VaccinationSample\Program.cs(76,52): error CS0103: The name 'VaccinationCertificateFrame' does not exist in the current context [C:\TDW\repos\sdk-examples\dotnet\VaccinationSample\VaccinationSample.csproj]
C:\TDW\repos\sdk-examples\dotnet\VaccinationSample\Program.cs(92,15): error CS1061: 'WalletService' does not contain a definition for 'Options' and no accessible extension method 'Options' accepting a first argument of type 'WalletService' could be found (are you missing a using directive or an assembly reference?) [C:\TDW\repos\sdk-examples\dotnet\VaccinationSample\VaccinationSample.csproj]
C:\TDW\repos\sdk-examples\dotnet\VaccinationSample\Program.cs(99,1): error CS0103: The name 'Assert' does not exist in the current context [C:\TDW\repos\sdk-examples\dotnet\VaccinationSample\VaccinationSample.csproj]
    0 Warning(s)
    16 Error(s)
```

--- a/dotnet/VaccinationSample/Program.cs
+++ b/dotnet/VaccinationSample/Program.cs
@@ -1,27 +1,30 @@
 ﻿using System;
 using System.IO;
-using Newtonsoft.Json;
-using Newtonsoft.Json.Linq;
+using System.Linq;
+using FluentAssertions;
 using Trinsic;
+using Trinsic.Sdk.Options.V1;
+using Trinsic.Services.Account.V1;
+using Trinsic.Services.Provider.V1;
+using Trinsic.Services.UniversalWallet.V1;
+using Trinsic.Services.VerifiableCredentials.V1;
+
+const string VaccinationCertificateUnsigned = "vaccination-certificate-unsigned.jsonld";
+const string VaccinationCertificateFrame = "vaccination-certificate-frame.jsonld";
 
 // createAccountService() {
-var _options = new ServiceOptions() {
-    ServerEndpoint = "prod.trinsic.cloud"
-    ServerPort = 443
-    ServerUseTls = true
-};
-var providerService = new ProviderService(_options.Clone());
-var accountService = new AccountService(_options.Clone());
-var (ecosystem, _) = providerService.CreateEcosystem(new());
+var providerService = new ProviderService();
+var accountService = new AccountService();
+var (ecosystem, _) = providerService.CreateEcosystem(new CreateEcosystemRequest());
 var ecosystemId = ecosystem.Id;
 // }
 
 // SETUP ACTORS
 // Create 3 different profiles for each participant in the scenario
 // setupActors() {
-var allison = await accountService.SignInAsync(new() {EcosystemId = ecosystemId});
-var clinic = await accountService.SignInAsync(new() {EcosystemId = ecosystemId});
-var airline = await accountService.SignInAsync(new() {EcosystemId = ecosystemId});
+var allison = await accountService.SignInAsync(new SignInRequest { EcosystemId = ecosystemId });
+var clinic = await accountService.SignInAsync(new SignInRequest { EcosystemId = ecosystemId });
+var airline = await accountService.SignInAsync(new SignInRequest { EcosystemId = ecosystemId });
 // }
 
 accountService.Options.AuthToken = clinic;
@@ -29,8 +32,8 @@ var info = await accountService.GetInfoAsync();
 info.Should().NotBeNull();
 
 // createService() {
-var walletService = new WalletService(_options.CloneWithAuthToken(allison));
-var credentialsService = new CredentialsService(_options.CloneWithAuthToken(clinic));
+var walletService = new WalletService(accountService.Options.CloneWithAuthToken(allison));
+var credentialsService = new CredentialsService(accountService.Options.CloneWithAuthToken(clinic));
 // }
 
 // ISSUE CREDENTIAL
@@ -43,7 +46,7 @@ walletService.Options.AuthToken = credentialsService.Options.AuthToken = clinic;
 // Read the JSON credential data
 var credentialJson = await File.ReadAllTextAsync(VaccinationCertificateUnsigned);
 // Sign the credential using BBS+ signature scheme
-var credential = await credentialsService.IssueCredentialAsync(new() {DocumentJson = credentialJson});
+var credential = await credentialsService.IssueCredentialAsync(new IssueRequest { DocumentJson = credentialJson });
 Console.WriteLine($"Credential:\n{credential.SignedDocumentJson}");
 // }
 
@@ -60,9 +63,9 @@ allison = File.ReadAllText("allison.txt");
 // Set active profile to 'allison' so we can manage her cloud wallet
 walletService.Options.AuthToken = credentialsService.Options.AuthToken = allison;
 
-var itemId = await walletService.InsertItemAsync(new() {ItemJson = credential.SignedDocumentJson});
+var itemId = await walletService.InsertItemAsync(new InsertItemRequest { ItemJson = credential.SignedDocumentJson });
 var walletItems = await walletService.SearchAsync();
-Console.WriteLine($"Last wallet item:\n{walletItems.Items.Last()}");
+Console.WriteLine($"Last wallet item:\n{walletItems.Items.ToList().Last()}");
 // }
 
 // SHARE CREDENTIAL
@@ -77,7 +80,8 @@ var proofRequestJson = await File.ReadAllTextAsync(VaccinationCertificateFrame);
 
 // Build a proof for the given request and the `itemId` we previously received
 // which points to the stored credential
-var credentialProof = await credentialsService.CreateProofAsync(new() {
+var credentialProof = await credentialsService.CreateProofAsync(new CreateProofRequest
+{
     ItemId = itemId,
     RevealDocumentJson = proofRequestJson
 });
@@ -92,9 +96,20 @@ Console.WriteLine(credentialProof.ProofDocumentJson);
 walletService.Options.AuthToken = credentialsService.Options.AuthToken = airline;
 
 // Check for valid signature
-var valid = await credentialsService.VerifyProofAsync(new() {
+var valid = await credentialsService.VerifyProofAsync(new VerifyProofRequest
+{
     ProofDocumentJson = credentialProof.ProofDocumentJson
 });
 Console.WriteLine($"Verification result: {valid}");
-Assert.True(valid);
+valid.Should().BeTrue();
 // }
+
+static class Extensions
+{
+    public static ServiceOptions CloneWithAuthToken(this ServiceOptions options, string authToken)
+    {
+        var cloned = options.Clone();
+        cloned.AuthToken = authToken;
+        return cloned;
+    }
+}

--- a/dotnet/VaccinationSample/VaccinationSample.csproj
+++ b/dotnet/VaccinationSample/VaccinationSample.csproj
@@ -1,21 +1,21 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <OutputType>Exe</OutputType>
-    <TargetFramework>net5.0</TargetFramework>
-  </PropertyGroup>
+    <PropertyGroup>
+        <OutputType>Exe</OutputType>
+        <TargetFramework>net6.0</TargetFramework>
+    </PropertyGroup>
 
-  <ItemGroup>
-    <PackageReference Include="Okapi.Net" Version="1.0.1-preview.1" />
-    <PackageReference Include="Trinsic" Version="1.0.0-preview.3" />
-  </ItemGroup>
+    <ItemGroup>
+        <PackageReference Include="FluentAssertions" Version="6.2.0"/>
+        <PackageReference Include="Trinsic" Version="1.4.0"/>
+    </ItemGroup>
 
-  <ItemGroup>
-    <None Update="vaccination-certificate-unsigned.jsonld">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </None>
-    <None Update="vaccination-certificate-frame.jsonld">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </None>
-  </ItemGroup>
+    <ItemGroup>
+        <None Update="vaccination-certificate-unsigned.jsonld">
+            <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+        </None>
+        <None Update="vaccination-certificate-frame.jsonld">
+            <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+        </None>
+    </ItemGroup>
 </Project>

